docs/c8ql/functions/search.md

```diff
@@ -187,7 +187,7 @@ Match documents where the attribute at **path** is present _and_ was indexed by
 
 - **path** (attribute path expression): the attribute to test in the document
 - **type** (string): string literal `"analyzer"`
-- **analyzer** (string, _optional_): name of an [Analyzer](../../search/analyzers.md). Uses the Analyzer of a wrapping `ANALYZER()` call if not specified or defaults to `"identity"`
+- **analyzer** (string, _optional_): name of an [Analyzer](../../search/Analyzers.md). Uses the Analyzer of a wrapping `ANALYZER()` call if not specified or defaults to `"identity"`
```

Comment:
All filenames are lowercase. Please back this one out.

Reply:
And all others that change lowercase to uppercase.

Reply:
Try updating from master. Those uppercase files were changed to lowercase last week.

Reply:
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/107413064/179540088-0be20529-cb67-44f8-bd3e-1dd8a3cb7181.png">

Reply:
@diana-macrometa it shouldn't be a cache issue if you are running it locally cc @koshy-thomas

Reply:
cc @elof @diana-macrometa